style.css

```
@@ -0,0 +1,58 @@
+* {
+    margin: 0;
+    box-sizing: border-box;
+}
+
+html {
+    font-size: 10px;
+}
+
+header {
+    background: gray;
+    height: 100px;
+}
+
+main {
+    display: flex;
+    height: 990px;
+
+}
+
+main .main-nav {
+    float: left;
+    background: green;
+    flex: 1;
```

you can remove `float` and `flex` properties and just  set `width: 300px`, so it can stay in fixed width while screen resizes.

---

also you can add:

```suggestion
  display: flex;
  flex-direction: column;
  padding: 20px;
```

for better controlling of the sidebar